### DISCUSSION TITLE
[Q] Is the project live?

### DISCUSSION THREAD

**Discussion by gaydenko** [MobiusSync/MobiusSync#78] [General]
Hi @sjpickup ! 

Will you please be so kind to clarify current project status? At the moment iOS app version does support syncthing version two step behind from the upstream (and 7 month aged).

**Comment by lmcj-xyz** [comment]
Same question here, thinking of using an IPhone (getting the unlimited file sync of course) but not sure because the last version in the app store is from Nov 2021.

**Comment by kwlorenz** [comment]
No answer, no updates? Is it alive or not?

**Comment by SimonPickup** [comment]
I've been busy on other projects, but this is definitely not abandoned. If there is anything in particular you are missing from upstream Syncthing, please open a seperate issue and we will triage. Otherwise we expect to update the core at a minimum every 12 months.
